test/src/test/integration/librad/scenario/collaboration.rs

```diff
@@ -80,13 +80,13 @@ fn can_add_maintainer() {
                 move |storage| -> Result<Option<identities::VerifiedProject>, anyhow::Error> {
                     let project = identities::project::get(&storage, &rad)?.unwrap();
                     identities::project::update(
-                        &storage,
+                        storage,
                         &urn,
```

Comment:
Why doesn't `&urn` change to `urn` here? :thinking:

Reply:
AFAIU the closure that is passed to `using_storage` is of `FnOnce(&git::storage::Storage) -> A + Send + 'static` which means `storage` is is already a reference and before it read `&&storage` which is immediately dereferenced to `&storage` as per the expected signature of `identities::project::update`.

https://github.com/radicle-dev/radicle-link/blob/138813dba6d2cf377865ba163049982cb4c03c1c/librad/src/net/peer.rs#L248

https://github.com/radicle-dev/radicle-link/blob/138813dba6d2cf377865ba163049982cb4c03c1c/librad/src/git/identities/project.rs#L124-L130

Reply:
Ohhhh that makes sense :) Thanks for clearing that up!